src/routes/safe/components/Apps/confirmTransactions.js

```diff
@@ -38,9 +48,31 @@ const confirmTransactions = (
   closeModal: () => void,
   onConfirm: () => void,
 ) => {
+  const isTxError = txs.some((t) => {
```

Comment:
I would rename `isTxError` to `isValidTx`, `isTxError` may be used later to check if there was an error when performing a transaction

Reply:
done!